docs/FAQ.md

```diff
@@ -63,11 +63,11 @@ Our VM and language are really dev-friendly and if you're familiar with Solidity
 
 ### Where is the roadmap?
 
-You will find it on the Wiki: https://wiki.alephium.org/#roadmap and on the website: https://alephium.org/#what's_next. You can also follow our weekly development updates on Discord, Twitter or Reddit.
+You will find it on the Wiki: https://docs.alephium.org/#roadmap and on the website: https://alephium.org/#what's_next. You can also follow our weekly development updates on Discord, Twitter or Reddit.
 
 ### Where can i query an API?
```

Comment:
Not linked to this PR, but 
can I query... "i" should be uppercase 🙃

Reply:
merci! fixed in https://github.com/alephium/wiki/pull/118/commits/cc648b062452f7d9350461a5addb712474bb4446